day_7/caesar_cipher.rb

```diff
@@ -0,0 +1,11 @@
+puts "What word will we be ciphering today? "
+word = gets.chomp.downcase
+puts "What is your lucky number? "
+num = gets.chomp.to_i
+def caesar_cipher(word, num)
+alphabet = ('a'..'z').to_a
+key = Hash[alphabet.zip(alphabet.rotate(num))]
+word.each_char.inject("") { |newtext, char| newtext + key[char] }
+end
+
+puts "#{caesar_cipher(word, num)} is your new secret phrase!"
```

Comment:
This is a very elegant solution. As you modify and use code for your solutions, remember to check formatting (indentations on methods), and take time to research the methods used. Be sure to check out the `zip` and `inject` mythical creature ruby-exercises.